sklearn/utils/tests/test_extmath.py

```diff
@@ -568,6 +569,22 @@ def max_loading_is_positive(u, v):
     assert not v_based
 
 
+def test_randomized_svd_parametrization():
+    # Check that function is correctly parametrized
```

Comment:
What I mean is to write a test to provide "gesdd" and "gesvd" and check that we get some results.

Reply:
A test that calls the `randomized_svd` function with both `lapack_svd_driver="gesdd"` and `lapack_svd_driver="gesvd"` and check that the results are approximately the same (up to rounding errors) using `assert_allclose`.